Gemfile.lock

```diff
@@ -514,7 +517,7 @@ GEM
     omniauth-twitter (1.4.0)
       omniauth-oauth (~> 1.1)
       rack
-    openstax_active_force (1.1.0)
+    openstax_active_force (1.1.1)
```

Comment:
@Dantemss I went ahead and upgraded this as part of this PR.
This is all that's needed, right? No update to `openstax_salesforce` required?